routing/dht/records.go

```diff
@@ -43,20 +43,20 @@ func RecordBlobForSig(r *pb.Record) []byte {
 }
 
 // creates and signs a dht record for the given key/value pair
-func (dht *IpfsDHT) makePutRecord(key u.Key, value []byte) (*pb.Record, error) {
+func MakePutRecord(sk ci.PrivKey, key u.Key, value []byte) (*pb.Record, error) {
```

Comment:
yeah let's pull out the record stuff into `routing/record` so ipns can import `record` and not `dht`.